etcdmain/help.go

```diff
@@ -206,6 +206,8 @@ Experimental feature:
     ExperimentalEnableLeaseCheckpoint enables primary lessor to persist lease remainingTTL to prevent indefinite auto-renewal of long lived leases.
   --experimental-compaction-batch-limit
     ExperimentalCompactionBatchLimit sets the maximum revisions deleted in each compaction batch.
+  --experimental-peer-skip-client-san-verification
```

Comment:
@jingyih thanks and small observation, since it's a boolean we should set it with default `false` similar to other such flags?. Also, on a side note I noticed that `--experimental-enable-lease-checkpoint` is a boolean and should set with its default `false` as well  - if that makes sense, may be you want to update that here as well here? Thanks!

Reply:
Thanks! Looks like there are several default values missing in the help message. Updated. PTAL.

Reply:
Awesome!! Thanks @jingyih 
LGTM

Reply:
Thanks @spzala !